apis/kubedb/v1alpha1/mariadb_helpers.go

```diff
@@ -173,3 +174,9 @@ func (m *MariaDBSpec) GetSecrets() []string {
 	}
 	return secrets
 }
+
+func (m *MariaDB) IsReplicasReady(stsLister *appslister.StatefulSetLister) (bool, string, error) {
```

Comment:
This does not go here. The "api" package should not depend on the "client" .

Reply:
Why we will need client here? We have passed lister. Can't we list from the lister without client?